src/tests/ut/visit/visit_test.cpp

```diff
@@ -1,7 +1,10 @@
-#include "../app/doctor/doctor.hpp"
-#include "../app/visit/visit.hpp"
+#include "doctor.hpp"
+#include "visit.hpp"
 #include "gtest/gtest.h"
 
+namespace unit_test::visit
```

Comment:
This isn't a scope of this change, but could you create a header file  and move structure over there and keep tests here with anonymous namespace?

Reply:
done